CMakeLists.txt

```
@@ -91,6 +91,9 @@ SET(${PROJECT_NAME}_ENABLE_CPACK_PACKAGING_DEFAULT ON)
 # Don't allow disabled subpackages to be excluded from tarball
 SET(${PROJECT_NAME}_EXCLUDE_DISABLED_SUBPACKAGES_FROM_DISTRIBUTION_DEFAULT FALSE)
 
+# See https://docs.trilinos.org/files/TrilinosBuildReference.html#setting-the-install-prefix
+SET(Trilinos_USE_GNUINSTALLDIRS ON)
```

```suggestion
SET(Trilinos_USE_GNUINSTALLDIRS_DEFAULT ON)
```

As per:

* https://tribitspub.github.io/TriBITS/users_guide/index.html#project-name-use-gnuinstalldirs

this has to set the non-cache var `Trilinos_USE_GNUINSTALLDIRS_DEFAULT` **not** `Trilinos_USE_GNUINSTALLDIRS`.